modules/ossm-remove-cleanup-1x.adoc

```diff
@@ -37,7 +37,7 @@ $ oc delete mutatingwebhookconfigurations/openshift-operators.servicemesh-resour
 +
 [source,terminal]
 ----
-$ oc delete -n openshift-operators daemonset/istio-node
+$ oc delete -n openshift-operators daemonset/istio-cni-node
```

Comment:
Is this file (ossm-remove-cleanup-1x.adoc) part of the docs for the v1.x version of the *operator* or for when you deploy a v1.x control plane via the v2.x operator? If the latter, then the DaemonSet name must remain `istio-node`.

Reply:
@luska, see what I get for trying to be proactive?    :)

Thanks for commenting on this. I've updated the PR with the change.